Polyfills/Canvas/Source/Image.cpp

```diff
@@ -91,22 +94,51 @@ namespace Babylon::Polyfills::Internal
         request.SendAsync().then(m_runtimeScheduler, *m_cancellationSource, [env{info.Env()}, this, request{std::move(request)}](arcana::expected<void, std::exception_ptr> result) {
             if (result.has_error())
             {
+                if (!m_onerrorHandlerRef.IsEmpty())
+                {
+                    m_onerrorHandlerRef.Call({});
```

Comment:
Why is an empty object passed here?

Reply:
Yes we should be returning thanks!

Added the error object to the callback params.